app/models/connector.rb

```diff
@@ -2,15 +2,11 @@
 #
 # Table name: connectors
 #
-#  id             :integer          not null, primary key
-#  resource_id_id :integer
-#  topic_id_id    :integer
-#
-# Indexes
-#
-#  index_connectors_on_resource_id_id  (resource_id_id)
-#  index_connectors_on_topic_id_id     (topic_id_id)
+#  id          :integer          not null, primary key
```

Comment:
I like how you define the structure of the table in comments at the top of the file, it really helps in understanding the structure and the codebase quickly

Reply:
Agreed, I think these comments help new viewers understand the structure of the table quickly and can also serve as a refresher for people who haven't viewed the file in a while.